tests/test_domains/test_domain_py.py

```diff
@@ -967,6 +967,133 @@ def test_domain_py_python_maximum_signature_line_length_in_text(app):
     assert expected_parameter_list_foo in content
 
 
+@pytest.mark.sphinx(
```

Comment:
This is not very elegant, but I don't think there is a way to parametrize a confoverride? If this can somehow be merged in the test above in a smart way, let me know though!

Reply:
You can parametrize the confoverrides and manually create the app itself but I think it's fine that way. Those tests are for two different purposes IMO and it's clearer that way.